eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -20,6 +20,7 @@ module Cardano.Ledger.Conway.Rules.Ratify (
   spoAcceptedRatio,
   dRepAccepted,
   dRepAcceptedRatio,
+  acceptedByEveryone,
```

Comment:
If you wanted you could add to conway CHANGELOG
Added `showGovActionType`  and  `acceptedByEveryone`

Reply:
Yup, thanks for reminding